quarkToJson.scd

```diff
@@ -7,7 +7,7 @@ q[\toJson] = {|v|
 	{v.class == Association} { q[\toJson].value(v.key) }
 	{v===true} { "true" }
 	{v===false} { "false" }
-	{v.isString} { "\"%\"".format(v.replace("\"", "\\\"")).replace("\n", "\\n").replace("\t", " ") }
+	{v.isString} { "\"%\"".format(v.asString.replace("\"", "\\\"")).replace("\n", "\\n").replace("\t", " ") }
```

Comment:
have you swapped lines by accident? The `v.isString` case assures that `v` is a string, the `Symbol` case below misses the explicit conversion into a String.

Reply:
whoops! yeah, i used the web UI instead of my local copy to make the PR. One sec

Reply:
updated